README.md

```diff
@@ -19,34 +19,42 @@ then you will have the extensions added to the site.
 
 To use these extensions on the site, on the Civi menu on the site go to administer - customise data and screens - manage extensions. There you should install CiviXero and Account Sync.
 
-You should now have a Xero tab in the civi menu. From here you can edit the xero settings. To do this 
-
-You need a Xero api key. 
+SETUP XERO KEYS
 
 Log into https://api.xero.com/Application?redirectCount=0
 
 Choose My Applications, Add application
-<img src='https://raw.githubusercontent.com/eileenmcnaughton/nz.co.fuzion.civixero/master/docs/images/create_application.png'>
+<img src='https://github.com/kayliefz/nz.co.fuzion.civixero/blob/master/docs/images/Add_new_App.png'>
```

Comment:
We just need to fix this back now - I'll merge first because it will probably make it easier to do the follow up